tools/docker/daml-all-in-one/Readme.md

```diff
@@ -0,0 +1,30 @@
+# DAML All in One Image
```

Comment:
@raynatopedrajeta Sorry for the annoying nit-pick: Please call the file `README.md` just to keep with the conventions.

Reply:
@raynatopedrajeta Same request again. ^^

Reply:
@raynatopedrajeta Same request again. ^^